spec/system/user_views_logo_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+
+RSpec.describe "Logo behaviour with creator_onboarding Feature Flag", type: :system do
```

Comment:
This is a temporary spec to determine the behaviour of the logo based on the Feature Flag. It will mos likely be removed. I did it in capybara instead of Cypress because we haven't yet figured out how to toggle feature flags in Cypress. Also this test will be short lived.